format/testfiles/test.rego.formatted

```diff
@@ -39,9 +39,7 @@ partial_set["y"]
 r = y {
 	y = x
 	split("foo.bar", ".", input.x) with input as {"x": x}
-}
-
-# Comment on else
+} # Comment on else
 else = y {
```

Comment:
Hmm it seems like maybe if there are comments on the else like this we should maybe keep the new line separation.

Personal preference maybe, but IMO:

```rego
# base case for p
p = x {
    ...
}

# some special case
# with lots of comments
# describing it
else = y {

}
```

is nicer to read than
```rego
# base case for p
p = x {
    ...
} # some special case
# with lots of comments
# describing it
else = y {

}
```
Thoughts?

I'm wondering too if we should try and respect the original line breaks, like if the else was "inlined" or on a new line originally we format it to stay in that style. We do some checks for that other places in the formatter https://github.com/open-policy-agent/opa/blob/master/format/format.go#L306 (although I'm not sure how effectively we can do the same here, just brain storming a little bit).